src/dtoannotations/Validator.java

```diff
@@ -0,0 +1,70 @@
+package dtoannotations;
+
+import java.lang.reflect.Field;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Validator {
+
+    public List<String> validate(Object object) throws IllegalAccessException {
+        List<String> list = new ArrayList<>();
+        Class<?> objectClass = object.getClass();
+        Field[] declaredFields = objectClass.getDeclaredFields();
+        for (Field declaredField : declaredFields) {
+            Object fieldValue = declaredField.get(objectClass);
+            if (declaredField.isAnnotationPresent(Length.class)) {
+                Length length = declaredField.getAnnotation(Length.class);
+                if (fieldValue.toString() != null) {
```

Comment:
No need to use 2 IF statements, you can use **&&** operator.

Reply:
done